src/types.ts

```diff
@@ -19,6 +19,20 @@ export type BundleEntry = {
    * Defaults to `dist/` if not provided.
    */
   outDir?: string;
+
+  /**
+   * Minify the output via oxc-minify.
+   *
+   * Defaults `false` if not provided.
+   */
+  minify?: boolean;
+
+  /**
+   * Generate and bundle dts files via rolldown-plugin-dts.
+   *
+   * Defaults `false` if not provided or `isolatedDeclarations` in `compilerOptions` is set to `false`.
+   */
+  declaration?: boolean;
```

Comment:
We need to respect both also in `transform.ts`

Reply:
L43 and L138 in `transform.ts, ' right? I have time now, and let me resolve this